### DISCUSSION TITLE
Saving data icon is spinning image of locally logged in discord user when possible instead of default one

### DISCUSSION THREAD

**Discussion by JezSonic** [NewDEV-github/Foxy-Adventure#47] [Ideas] [answered]
So, i was thinking about making an saving data icon and i thought: `We have discord game sdk wich can get current discord user profile pic and return it to godot so if theres someone logged in on discord, sdk can get his image and set it as saving data icon wich will have just additional animation made in godot and when there's no discord user logged in, there will be default saving data icon`

**Comment by infevlol** [comment]
what

**Comment by LennyPhoenix** [comment, marked as answer]
That should work fine, make sure you cache the image so you don't have to fetch it every time you enter a loading screen

**Reply by JezSonic** [reply]
thanks for advice 😄
